subsys/bluetooth/controller/ll_sw/ull_conn.c

```diff
@@ -4086,6 +4086,21 @@ static inline void event_phy_upd_ind_prep(struct ll_conn *conn,
 		struct node_rx_pdu *rx;
 		uint8_t old_tx, old_rx;
 
+		/* Acquire additional rx node for Data length notification as
+		 * a peripheral.
+		 */
+		if (IS_ENABLED(CONFIG_BT_PERIPHERAL) &&
+		    IS_ENABLED(CONFIG_BT_CTLR_DATA_LENGTH) &&
+		    conn->lll.role) {
+			rx = ll_pdu_rx_alloc();
+			if (!rx) {
+				return;
+			}
```

Comment:
What happens if it cannot be allocated? Is it okay that it fails silently like this?

Reply:
No action has been taken, and an attempt will be done in successive connection events. If there is buffer starvation (which should not happen), the connection will terminate eventually by supervision timeout due to PHY update procedure being not complete after the instant. This is the change compared to the earlier fatal assert.